### DISCUSSION TITLE
Unable to install duckdns plugin

### DISCUSSION THREAD

**Discussion by abhinavisgood** [lxhao61/integrated-examples#35] [Q&A] [answered]

**Comment by lxhao61** [comment, marked as answer]
下载最新（刚更新）的caddy文件。

**Comment by abhinavisgood** [comment]
Thank you very  much

**Comment by abhinavisgood** [comment]
One More Question , With Caddy(N+T) , is proxied Option ON in Cloudflare DNS panel advisable as  ON , because with that its not able to access the endpoint

**Comment by abhinavisgood** [comment]
I want to employ CDN also but am unable to do so with the freenom domains in cloudflare

**Comment by lxhao61** [comment]
> One More Question , With Caddy(N+T) , is proxied Option ON in Cloudflare DNS panel advisable as ON , because with that its not able to access the endpoint

只有使用 Cloudflare 提供的免费证书，面板中的CDN才必须开启；否则就是你设置出了问题。

**Comment by lxhao61** [comment]
> I want to employ CDN also but am unable to do so with the freenom domains in cloudflare

cloudflare 可以使用 freenom 域名，跟 CDN 无关！只是 freenom 域名不能以 DNS-01 验证方式申请证书而已（仔细看介绍，此部分信息已经告知。）！HTTP-01 或 TLS-ALPN-01 验证方式不影响！
